### PR TITLE
Fixed some bugs

### PR DESCRIPTION
**Pull request by pedroaugustobt**
Sterilizing Fluid has this extra effect in addition to the healing that is already built into the system.
image

Item effects were not appearing, and hands were not formatted according to the translations.
image

### CODE REVIEW

File: module/actor/sheets/WitcherActorSheet.js [hunk 1]
```diff
@@ -337,6 +337,10 @@ export default class WitcherActorSheet extends ActorSheet {
                         let newCritList = [];
                         critList.forEach(crit => {
                             crit.daysHealed += 1;
+                            // Reduce the healing time by 2 days if sterFluid was used
+                            if (sterFluid) {
```
Comment (Stexinator) [inline module/actor/sheets/WitcherActorSheet.js:341]: This is already factored into the healing time line 309

**Review by Stexinator — CHANGES_REQUESTED** [review]

File: module/actor/sheets/WitcherActorSheet.js (hunk 1 above)
Comment (pedroaugustobt) [reply module/actor/sheets/WitcherActorSheet.js:341]: The code is specifically designed to decrease the maximum recovery time for a critical injury by 2 days whenever a specific item is used

Comment (Stexinator) [reply module/actor/sheets/WitcherActorSheet.js:341]: The code at line 309 adds 2 days when it is used. So this does the same

Comment (pedroaugustobt) [reply module/actor/sheets/WitcherActorSheet.js:341]: ah, ok. I will remove this code

Comment (Stexinator) [reply module/actor/sheets/WitcherActorSheet.js:341]: I also already commited your fix for the path building in another feature I added today

Comment (pedroaugustobt) [reply module/actor/sheets/WitcherActorSheet.js:341]: ok

> I also already commited your fix for the path building in another feature I added today

Comment (pedroaugustobt) [reply module/actor/sheets/WitcherActorSheet.js:341]: I will remove these two commits

**Pull request merged**